avaframe/com1DFAPy/DFAfunctionsCython.pyx

```diff
@@ -282,18 +292,12 @@ def computeForceC(cfg, particles, fields, dem, Ment, Cres, dT):
 
       # adding bottom shear resistance contribution
       forceBotTang = - A * tau
-      forceFrict[j] = forceFrict[j] - forceBotTang
+
+      forceFrict[j] =  forceFrict[j] - forceBotTang / uMag
```

Comment:
I do not like the naming... It is called force but it is not a force!!! it is not in Newtons... Peter calls this resistance, I do not know what is the best naming